.travis.yml

```diff
@@ -28,6 +28,7 @@ matrix:
       env: JRUBY_OPTS="--debug -X+O"
     - rvm: truffleruby
     - rvm: ruby-head
+    - rvm: git@github.com:ioquatix/ruby#thread-selector
```

Comment:
How can I make this work?

Reply:
I _think_ it'd be:

```yaml
- rvm: ruby-head-scheduler --url "https://github.com/ioquatix/ruby.git" --branch selector
```
P.S. - I meant to push a commit to my fork of your brach to test CI, not your branch itself. Reverted. I'm sorry!

Reply:
Nope, I'm wrong. It doesn't work since Travis is appending a `--binary` flag. https://travis-ci.org/havenwood/async/jobs/635745650

Reply:
Since this is now merged into master, this is no longer a problem *for this PR*. Thanks for your help.